docs(getting-started): Overhaul (#2754)
kubeshop/tracetest: docs/docs/getting-started/cli-installation-reference.md

# CLI Installation Reference

> This page contains a reference of all options how to install Tracetest CLI.

Reviewer: ```suggestion
This page contains a reference of all options for installing Tracetest CLI.
```
